### PR TITLE
feature: enable decision publishing

### PR DESCRIPTION
**Pull request by hsiliev**
Enable publishing for indienen aansprakelijkstelling zaaktype decisions

Solves: PZ-4804

### CODE REVIEW

**Review by xiduzo — APPROVED** [review]

**Review by xiduzo — COMMENTED** [review]
due to my lack of knowledge of the system, I can not really see the implications of this change

File: scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql [hunk 1]
```diff
@@ -902,10 +902,10 @@ VALUES
   'Besluit aansprakelijkstelling',
   'besluit-aansprakelijkstelling',
   'besluit',
+  'P2D',
```
Comment (edgarvonk) [inline scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql:905]: Maybe add a code comment here? Nobody will know what this does in a year's time I'm afraid.. ;-)

**Review by edgarvonk — APPROVED** [review]
Small comment

File: scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql (hunk 1 above)
Comment (hsiliev) [reply scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql]: Done. 

This should be documented on OpenZaak side as this format is enforced by them
On our side we use java.time.Period to parse these strings

Comment (edgarvonk) [reply scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql]: I know but this is our code base so we should also document for our future developers. :-)

**Pull request merged**